server/src/test/scala/org/broadinstitute/dsp/workbench/welder/server/StorageLinksApiServiceSpec.scala

```diff
@@ -0,0 +1,82 @@
+package org.broadinstitute.dsp.workbench.welder
+package server
+
+import java.nio.file.Paths
+
+import _root_.io.chrisdavenport.log4cats.Logger
+import _root_.io.chrisdavenport.log4cats.slf4j.Slf4jLogger
+import cats.effect.IO
+import cats.effect.concurrent.Ref
+import cats.implicits._
+import org.http4s.circe.CirceEntityEncoder._
+import fs2.text
+import io.circe.{Json, parser}
+import org.broadinstitute.dsde.workbench.model.google.GcsBucketName
+import org.broadinstitute.dsp.workbench.welder.LocalBasePath.{LocalBaseDirectoryPath, LocalSafeBaseDirectoryPath}
+import org.http4s.{Method, Request, Status, Uri}
+import org.scalatest.FlatSpec
+
+class StorageLinksApiServiceSpec extends FlatSpec with WelderTestSuite {
+  implicit val unsafeLogger: Logger[IO] = Slf4jLogger.getLogger[IO]
+  val storageLinks = Ref.unsafe[IO, Map[LocalBasePath, StorageLink]](Map.empty)
+  val storageLinksService = StorageLinksService(storageLinks)
+  val cloudStorageDirectory = CloudStorageDirectory(GcsBucketName("foo"), BlobPath("bar"))
+  val baseDir = LocalBaseDirectoryPath(Paths.get("/foo"))
+  val baseSafeDir = LocalSafeBaseDirectoryPath(Paths.get("/bar"))
```

Comment:
would be nice if this can be changed to use [Generators](https://github.com/DataBiosphere/welder/pull/10/files#diff-77f8be254dd22acdad929b4d80cb0f71R34)